### PR TITLE
prometheus service auto-discovery  

### PR DESCRIPTION
**Pull request by SuleimanWA**
*#1*

**Describe your changes**
goldPinger expose prometheus metrics at ip:port/metrics, and you need to send it to prometheus or add configuration to prometheus in order to fetch, but if this request merged, you will allow prometheus to auto-discover your metrics and fetch it from service-name:ip/metrics 

**Testing performed**
I add this and checked from prometheus side if target is up and metrics fetched.

**Additional context**
Add any other context about your contribution here.

### CODE REVIEW

**Comment by seeker89** [comment]
Excellent point! Please could you follow https://github.com/bloomberg/goldpinger/pull/71/checks?check_run_id=321407289 to sign off the commits so that I can approve and merge?

**Comment by seeker89** [comment]
Hey @SuleimanWA - every commit in the PR needs to have this `Signed off` part in the description.

Since it's a small change, it might be the easiest to just commit that on a new branch with `git commit -s` (`-s` adds the appropriate message).

Thanks!

**Comment by SuleimanWA** [comment]
@seeker89 a lot of things happened by mistake, sorry for this, is it in good shape now?

File: README.md [hunk 1]
```diff
@@ -191,6 +191,9 @@ kind: Service
 metadata:
   name: goldpinger
   namespace: default
+  annotations:
+    prometheus.io/scrape: 'true'
+    prometheus.io/port: '8080'
```
Comment (seeker89) [inline README.md]: Just one note - if you add the annotations on the service, then only one pod will be randomly scraped each time. I would recommend to put it directly on the pods, somewhere around the line 138.

**Comment by seeker89** [comment]
No problem at all. I've left one comment, and then we should be good to go.

File: README.md (hunk 1 above)
Comment (SuleimanWA) [reply README.md]: But metrics exposed at url/metrics which is not pod specific right? it's for everything

Comment (SuleimanWA) [reply README.md]: what I mean we don't care about fetch from all, we care about fetch from anyone at any point of time

Comment (seeker89) [reply README.md]: No, each pod serves the metrics from its POV. When producing the graph on the UI, we ask one pod to ask all other pods for their version of events, and then aggregate it for us in a single image. For metrics, on the other hand, each pod is advertising what it's seen and then it's up to your grafana kung-fu to aggregate, slice and dice and process it.

Comment (SuleimanWA) [reply README.md]: annotations moved from service to pod and tested

**Pull request merged**